File: sslvagrant/Week1/grader.rb
class Grader
    def grader(_grade, _student, _assignment) 
        grade = _grade.to_f
        puts('-------------------------------------')
        puts('Student Name: ' + _student)
        puts('Assignment Name: ' + _assignment)
        if grade < 60 
            puts ("Grade: F")
        elsif grade >= 60 && grade <= 69 
            puts ("Grade: D")
        elsif grade >= 70 && grade <= 79 
            puts ("Grade: C")
        elsif grade >= 80 && grade <= 89 
            puts ("Grade: B")
        else
            puts ("Grade: A")
       
        end
        puts('-------------------------------------')
      
    end
    print "What is the name of the student? "
    student_name = gets.chomp
    print "What is the assignment's name? "
    assignment_name = gets.chomp
    print "What's your grade: "
    grade = gets.chomp
    gradex = Grader.new
    gradex.grader(grade, student_name, assignment_name)
end

File: sslvagrant/Week1/grader.py
import sys
def grader(_grade, _student, _assignment):
	grade = float(_grade)
	print('-------------------------------------')
	print('Student Name: ' + _student)
	print('Assignment Name: ' + _assignment)
	if grade < 60:
		print ("Grade: F")
	elif grade >= 60 and grade <= 69:
		print ("Grade: D")
	elif grade >= 70 and grade <= 79:
		print ("Grade: C")
	elif grade >= 80 and grade <= 89:
		print ("Grade: B")
	else:
		print ("Grade: A")
	print('-------------------------------------')



student_name = raw_input("What is the name of the student? ")
assignment_name = raw_input("What is the assignment's name? ")
grade = raw_input("What's your grade: ")
grader(grade, student_name, assignment_name)
